src/ahp/constraint_systems.rs

```diff
@@ -20,33 +17,8 @@ use derivative::Derivative;
 /* ************************************************************************* */
 /* ************************************************************************* */
 
-pub(crate) fn balance_matrices<F: Field>(a_matrix: &mut Matrix<F>, b_matrix: &mut Matrix<F>) {
```

Comment:
we no longer need to balance matrices, as the cost is ~sum of non-zero entries across all matrices, as opposed to ~max of non-zero entries.

(While this seems worse, we actually performs smaller FFTs during proving now, and the degree of only one polynomial is slightly higher)